game-app/game-headed/src/main/java/games/strategy/triplea/ui/PurchasePanel.java

```diff
@@ -53,6 +54,12 @@ public class PurchasePanel extends ActionPanel {
         public void actionPerformed(final ActionEvent e) {
           final GamePlayer player = getCurrentPlayer();
           final GameData data = getData();
+          final PurchaseDelegate purchaseDelegate = data.getPurchaseDelegate();
+
+          // load pending production from file it there is any
```

Comment:
This comment isn't quite right, since we're not loading from file here - it was already loaded from file.

I think it should save something " // Restore pending production that was loaded from the save game."